File: 01_ProgrammingBasics/_09_BonusTasks/_01_DrawingFirguresWithLoops/_04_TriangleOfDollars.java
package _01_ProgrammingBasics._09_BonusTasks._01_DrawingFirguresWithLoops;

import java.util.Scanner;

public class _04_TriangleOfDollars {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        int n = Integer.parseInt(scan.nextLine());

        // Print a triangle of $ signs
        for (int i = 1; i <= n; i++) {
            for (int j = 1; j <= i; j++) {
                System.out.print("$ ");
            }
            
            System.out.println();
        }

        scan.close();
    }
}

File: 23_C#Basics/_07_BonusTasks/_01_DrawingFiguresWithLoops/_04_TriangleOfDollars/_04_TriangleOfDollars.cs
using System;

public class _04_TriangleOfDollars
{
    public static void Main(string[] args)
    {
        // Read input value
        int n = int.Parse(Console.ReadLine());

        // Print a triangle of $ signs
        for (int i = 1; i <= n; i++)
        {
            for (int j = 1; j <= i; j++)
            {
                Console.Write("$ ");
            }

            Console.WriteLine();
        }
    }
}
